docs/Lexicon.rst

```diff
@@ -471,7 +471,7 @@ source code, tests, and commit messages. See also the `LLVM lexicon`_.
     compared across translation units to determine, e.g., when references in
     one translation refer to an entity defined in another translation unit.
 
-  value witness table
+  VWT (value witness table)
```

Comment:
```suggestion
  value witness table (VWT)
```

Reply:
This would be inconsistent with the rest of the lexicon, where the abbreviation comes first. For example see below:

```
WMO (whole-module optimization)
```

or

```
vtable (virtual dispatch table)
```

Reply:
Fair point :)